### PR TITLE
enable contributions from forks

### PR DESCRIPTION
**Pull request by qrkourier**
These are the modifications I needed to run Actions so I could send a pull request.

### CODE REVIEW

**Comment by qrkourier** [comment]
@mattclegg Would you take a look at this one, please? I could not get the CI working in my fork with your changes, but the additional changes in this branch did work for me.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -54,6 +54,7 @@ jobs:
       - uses: chainguard-dev/actions/goimports@main
 
       - name: Login to registry
+        if: github.repository_owner == 'helm'
```
Comment (mattclegg) [inline .github/workflows/release.yaml:57]: Nice.

**Review by mattclegg — APPROVED** [review]
I can't confirm for the changes to [magefile.go](https://github.com/helm/chart-releaser/pull/440/files#diff-520109d035a196d29d0a86e9c4e6c98e98abf056141b84df68c0d48167b87b25) but the yml changes are correct.

@cpanato Are you able to check this please?

**Comment by qrkourier** [comment]
@mattclegg Would you please allow the checks to run?

@cpanato Please let me know if I can assist with further testing of explanations.

**Comment by mattclegg** [comment]
LGTM but I don't have permission for the checks to run as I'm not a maintainer.

**Comment by qrkourier** [comment]
@cpanato Would you like more explanation, examples, or testing for this?

File: .github/workflows/release.yaml (hunk 1 above)
Comment (cpanato) [inline .github/workflows/release.yaml:57]:
```suggestion
        if: github.repository == 'helm/chart-releaser'
```

File: magefile.go [hunk 2]
```diff
@@ -137,5 +137,12 @@ func Release() error {
 		return err
 	}
 
-	return sh.RunV("goreleaser", "release", "--clean")
+	var args []string
+	args = append(args, "release", "--clean")
+
+	if os.Getenv("GITHUB_REPOSITORY_OWNER") != "helm" {
```
Comment (cpanato) [inline magefile.go:143]: i would check for the entire things `helm/chart-releaser`

**Review by cpanato — CHANGES_REQUESTED** [review]
lgtm

just a few nits

**Review by cpanato — APPROVED** [review]

**Comment by cpanato** [comment]
thanks

**Pull request merged**